src/libostree/ostree-repo-checkout.c

```diff
@@ -35,6 +35,8 @@
 #define WHITEOUT_PREFIX ".wh."
 #define OPAQUE_WHITEOUT_NAME ".wh..wh..opq"
 
+#define OVERLAYFS_WHITEOUT_PREFIX ".wh-ostree."
```

Comment:
How about
```suggestion
#define OSTREE_TARGET_WHITEOUT_PREFIX ".ostree-wh."
```

Here "ostree_" is first because it's a thing conceptually namespaced/scoped to ostree, not to overlayfs.

Reply:
Done